### PR TITLE
syncer: configure filter

### PR DESCRIPTION
**Pull request by nolouch**
PTAL @siddontang @GregoryIan

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

PTAL @andelf

File: syncer/config.go [hunk 1]
```diff
@@ -96,6 +96,8 @@ type Config struct {
 
 	IgnoreDBs []string `toml:"replicate-ignore-db" json:"replicate-ignore-db"`
 
+	IgnoreSpecifySQLs []string `toml:"ignore-sqls" json:"ignore-sqls"`
```
Comment (siddontang) [inline syncer/config.go]: you can remove this now

**Comment by andelf** [comment]
Maybe it's better to combine patterns into a regexp, use "^(A|B|C)".

LGTM. 👻

**Comment by andelf** [comment]
and spaces should be: ``\s+``

for e.g.

```sql
alter\n\t\t      user ... blah
```

**Comment by siddontang** [comment]
@nolouch you can send another PR later

**Pull request merged**